server/db/models/User.js

```diff
@@ -74,12 +74,8 @@ User.byToken = async (token) => {
         const { id } = jwt.verify(token, process.env.JWT_SECRET);
         // TODO: separate logic
         // TODO: attributes to exclude - email, password, isAdmin
-        const user = await User.findByPk(id, {
-            include: ['friends', 'concerts'],
-        });
+        const user = await User.findUser(id);
```

Comment:
Since the `User.findUser` hook was created in a separate file, will we still have access in this class method?